src/swsssdk/configdb.py

```diff
@@ -193,6 +193,42 @@ def set_entry(self, table, key, data):
                     k = k + '@'
                 client.hdel(_hash, self.serialize_key(k))
 
+    def set_bulk(self, payload):
```

Comment:
>set_bulk [](start = 8, length = 8)

What are the use cases for the new functions? Application can achieve pipeline by calling get_redis_client().pipeline. And that solution will be more flexible since you can mix functions into the same pipeline.

Reply:
Qi, Please refer to the below comment from. It was suggested not to use the pipeline in the application.

https://github.com/Azure/sonic-utilities/pull/891#discussion_r420986191

The use case is also mentioned in the same PR. It is to create large number of VLANs. This also helps when querying and displaying the contents of a filled-up mac address table.

Reply:
Thanks @rajendra-dendukuri sharing the background! As we planed to move away from sonic-py-swsssdk and converge everything to sonic-swss-common, I believe new feature here will be short-lived. We have implemented some pipeline support in sonic-swss-common *Tables classes. Could you check if that fulfill your requirement?

Sorry for the inconvenience to extend code here.

---
In reply to: [445300357](https://github.com/Azure/sonic-py-swsssdk/pull/82#discussion_r445300357) [](ancestors = 445300357)